smart_control/environment/environment.py

```diff
@@ -1253,10 +1247,6 @@ def _action_strings(
     t0 = time.time()
     reward_value = 0.0
     observation = None
-    last_timestamp = self.current_simulation_timestamp
-
-    # Reformat actions if necessary.
-    action = self._format_action(action, self._action_names)
```

Comment:
removing invocation of no-op function that was removed (https://github.com/google/sbsim/pull/32/files#r2070820527)